internal/storage/storage.go

```diff
@@ -1,25 +1,91 @@
 package storage
 
 import (
+	"context"
+	"errors"
 	"hash/fnv"
+	"os"
 	"strconv"
 
+	"github.com/jackc/pgx/v5/pgxpool"
 	"github.com/webmstk/shorter/internal/config"
 )
 
 type Storage interface {
-	SaveLongURL(longURL string) (shortURL string, err error)
-	GetLongURL(shortURL string) (longURL string, ok bool)
+	SaveLongURL(ctx context.Context, longURL, userID string) (shortURL string, err error)
+	GetLongURL(ctx context.Context, shortURL string) (longURL string, ok bool)
```

Comment:
```suggestion
	GetLongURL(ctx context.Context, shortURL string) (longURL string, err error)
```
Лучше и правильней возвращать наверх ошибку, а не булево значение. В случае, если все хорошо, достаточно проверить на nil. Если все плохо (проблемы с файловой системой), можно узнать, что именно с файловой системой проблемы. Если возникает какое-то исключение, можно вернуть свой собственный err 
через `fmt.Error("")` или `errors.New("")`. По сути это есть стиль языка Go.

Reply:
понял, спасибо!